metatensor-torch/include/metatensor/torch/labels.hpp

```diff
@@ -25,6 +25,7 @@ namespace details {
     /// strings or a tuple of strings into a `std::vector<std::string>`.
     /// `argument_name` is used in the error message.
     std::vector<std::string> normalize_names(torch::IValue names, const std::string& argument_name);
+    torch::Device normalize_device(torch::IValue device);
```

Comment:
Please add a small docstring for this function, in particular explaining what's epxected in the `torch::IValue`